pipelines/main/platforms/build_macos.arches

```diff
@@ -0,0 +1,13 @@
+# OS       TRIPLET                 ALLOW_FAIL    ARCH_ROOTFS    MAKE_FLAGS     TIMEOUT       ROOTFS_TAG    ROOTFS_HASH
```

Comment:
No need for `ARCH_ROOTFS`, `ROOTFS_TAG` or `ROOTFS_HASH` here.  Those columns can be deleted.

Reply:
`ARCH_ROOTFS` should only be used with sandbox; any other uses of the arch should parse it out from the triplet, [which is done by `calc_version_envs.sh`](https://github.com/JuliaCI/julia-buildkite/blob/main/utilities/calc_version_envs.sh#L4)